### PR TITLE
Reactivates the CLF ERT

### PR DESCRIPTION
**Pull request by lbnesquik**
## About The Pull Request
This simple PR brings back the CLF. It was deactivated, but i do not see any reason why we shouldn't be able to see them.

## Why It's Good For The Game
More ERT's is fun.

## Changelog
:cl:
add: Brought back the CLF ERT to life.
/:cl:

### CODE REVIEW

**Comment by XSlayer300** [comment]
Wasn't already redundant as the CLF fufills the same purpose as the CLF?

Not to mention, the CLF was disabled because we separated ourselves from CM long time back.

**Comment by lbnesquik** [comment]
> 
> 
> Wasn't already redundant as the CLF fufills the same purpose as the CLF?
> 
> Not to mention, the CLF was disabled because we separated ourselves from CM long time back.

No clue, i didn't find a clear comment aobut it.

Also, i don't see what's really stopping us from using those ERT's, lore wise.

**Comment by SplinterGP** [comment]
Arent them past SoM? Pretty sure SOM replaced them.

**Comment by lbnesquik** [comment]
> 
> 
> Arent them past SoM? Pretty sure SOM replaced them.

I do not know the lore well enough to answer that. It would still be more variety regardless.

Actual worst case scenario, they get renamed in a different PR to something that fits better, If needed.

**Comment by TiviPlus** [comment]
They're different allied factions last I checked

**Review by TiviPlus — APPROVED** [review]

File: code/datums/emergency_calls/clf.dm [hunk 1]
```diff
@@ -1,7 +1,8 @@
 //Colonial Liberation Front
 /datum/emergency_call/clf
 	name = "CLF Cell"
-	base_probability = 0
+	base_probability = 20
+	alignement_factor = -1
```
Comment (TiviPlus) [inline code/datums/emergency_calls/clf.dm:6]: Actually thi should be 0 since clf hate both sides

**Review by TiviPlus — CHANGES_REQUESTED** [review]
.

File: code/datums/emergency_calls/clf.dm (hunk 1 above)
Comment (lbnesquik) [reply code/datums/emergency_calls/clf.dm]: Applied.

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**